index.js

```diff
@@ -53,7 +54,8 @@ module.exports = (input, output, options) => {
 	options = Object.assign({plugins: []}, options);
 	options.plugins = options.use || options.plugins;
 
-	return globby(input, {onlyFiles: true}).then(paths => Promise.all(paths.map(x => handleFile(x, output, options))));
+	return globby(input, {onlyFiles: true})
+		.then(paths => Promise.all(paths.reduce((acc, x) => junk.is(path.basename(x)) ? acc : acc.concat(handleFile(x, output, options)), [])));
```

Comment:
Using `reduce` here just makes the code much less readable. Would be better to use `filter`.